bazel/WORKSPACE

```diff
@@ -15,13 +15,43 @@
 # ==============================================================================
 
 workspace(name = "ngraph_bridge")
-load("@//tf_configure:tf_configure.bzl", "tf_configure")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+load("@//tf_configure:tf_configure.bzl", "tf_configure")
 tf_configure(
     name = "local_config_tf",
 )
 
+# TensorFlow depends on "io_bazel_rules_closure" so we need this here.
+# Needs to be kept in sync with the same target in TensorFlow's WORKSPACE file.
```

Comment:
We should document this requirement here (TF change/upgrade checklist): https://github.com/tensorflow/ngraph-bridge/wiki/Update-TF-version-in-ngraph-bridge

Reply:
Done